store/src/java/com/zimbra/cs/service/account/ChangePassword.java

```diff
@@ -108,6 +108,7 @@ public Element handle(Element request, Map<String, Object> context) throws Servi
         Element response = zsc.createElement(AccountConstants.CHANGE_PASSWORD_RESPONSE);
         if (!dryRun) {
            AuthToken at = AuthProvider.getAuthToken(acct);
+           at.setIgnoreSameSite(request.getAttributeBool(AccountConstants.A_IGNORE_SAME_SITE_COOKIE, false));
```

Comment:
if I remember correctly then changepassword just changes the password and doesn't update auth token, so why do we need this?